app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def update
 
   def create
     raise Exceptions::UnauthorizedError unless current_user.admin?
-    @user = User.new(user_params)
+    @user = User.new(user_params.merge(role: :cm))
```

Comment:
Could we get a comment here on what we're accomplishing by setting the role? `cm` is a pretty ab-fund-specific parlance, so I think some clarification of what CM means would be useful to someone seeing this for the first time who's less familiar.